opentasks/src/main/res/raw/sharetask.template

```diff
@@ -0,0 +1,18 @@
+{{ $task.title }}
+============
+{% if $task.description %}
```

Comment:
```
{% if $task.description %}
{{ $task.description }}
{% end %}
```
Can be replaced with
```
{{ $task.description or "" }}
```

Reply:
If I replace that then I can't make the formatting right when the description is not present. Too many empty lines. So I left this untouched.

Reply:
Are you sure? How would this issue in any blank lines if there is no description?
```
{{ $task.title }}
============
{{ $task.description or "" }}{% if $task.checklist %}{% for item in $task.checklist %}
[{% if item.checked %}X{% else %} {% end %}] {{ item.text }}{% end %}
{% end %}{% if $task.location %}
{{ $R.string.task_location}}: {{ $task.location }}{% end %}{% if $task.start %}
{{ $R.string.task_start}}: {{ tformat.apply($task.start) }}{% end %}{% if $task.due %}
{{ $R.string.task_due}}: {{ tformat.apply($task.due) }}{% end %}{% if $task.completed %}
{{ $R.string.task_completed}}: {{ tformat.apply($task.completed) }}{% end %}{% if $task.priority %}
{{ $R.string.task_priority}}: {{ $task.priority }}{% end %}{% if $task.privacy %}
{{ $R.string.task_classification}}: {{ $task.privacy }}{% end %}{% if $task.status %}
{{ $R.string.task_status}}: {{ $task.status }}{% end %}{% if $task.url %}
{{ $task.url }}{% end %}

--
{{ $R.string.opentasks_share_footer}}
```

This should result in exactly the same text.

Reply:
I've just tested this and it works for me.

Reply:
I've just tried and it produces different format that the original plan was, there is no empty line after the title, and also there is no empty line after the description, for me.

Wasn't this the plan?:
With description + checklist:
```
 <task title>
 ============

 <task description>

 [X] checked list item
 [ ] unchecked list item

 Location: <location>
 Start: <start date time> <timezone>
 Due: <due date time> <timezone>
 Completed: <due date time> <timezone>
 Priority: <priority text>
 Privacy: <privacy text>
 Status: <status text>
 <url>

 --
 Shared by OpenTasks
```
With checklist:
```
 <task title>
 ============

 [X] checked list item
 [ ] unchecked list item

 Location: <location>
 Start: <start date time> <timezone>
 Due: <due date time> <timezone>
 Completed: <due date time> <timezone>
 Priority: <priority text>
 Privacy: <privacy text>
 Status: <status text>
 <url>

 --
 Shared by OpenTasks
```
Without them:
```
 <task title>
 ============

 Location: <location>
 Start: <start date time> <timezone>
 Due: <due date time> <timezone>
 Completed: <due date time> <timezone>
 Priority: <priority text>
 Privacy: <privacy text>
 Status: <status text>
 <url>

 --
 Shared by OpenTasks
```
So to keep an empty lines always below the title.

Reply:
ah, right. Ok keep it like this then.